Dockerfile

```diff
@@ -26,4 +26,6 @@ FROM alpine:3.8
 ARG PROJECT_SLUG=github.com/tuenti/secrets-manager
 LABEL maintainer="sre@tuenti.com"
 COPY --from=builder /go/src/$PROJECT_SLUG/build/secrets-manager /secrets-manager
+RUN apk update
+RUN apk add ca-certificates
```

Comment:
Thanks for the contribution!

You could simplify this using a single line command and also add `--no-cache` and some post-install cleanup to avoid image bloat:

```
RUN apk add --update --no-cache ca-certificates && rm -rf /var/cache/apk/*
```

Reply:
Hi, thank you for optimization. I did the update :)